superset/assets/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -308,7 +308,7 @@ class SqlEditor extends React.PureComponent {
         </div>
         <SouthPane
           editorQueries={this.props.editorQueries}
-          latestQueryId={this.props.latestQuery ? this.props.latestQuery.id : 0}
+          latestQueryId={this.props.latestQuery && this.props.latestQuery.id}
```

Comment:
for brand new sql lab (no any query before, or user clears localStorage), the latestQuery object will be null. To fix this, i think 0 should be any string like '' or 'none' ?

Reply:
`null && something_invalid` resolves to null, so I think this is fine. It makes more sense for latestQueryId to be `null` if no query exists instead of something like empty string (i think)

Reply:
null is object type? The expected type is string.

Reply:
The expected type is string, not required, so null is fine right?

Reply:
oh just realized `latestQueryId` is optional. LGTM thanks!!